drivers/ieee802154/ieee802154_dw1000.c

```diff
@@ -1607,10 +1607,10 @@ static int dw1000_init(const struct device *dev)
 			  DWT_SYS_MASK_MRXSFDTO);
 
 	/* Initialize IRQ event work queue */
-	k_work_q_start(&dwt_work_queue,
+	k_work_queue_start(&dwt_work_queue,
```

Comment:
Not really related to the rework, but his work queue seems a bit suspicious, it doesn't actually seem to be used anywhere in the driver. CC @jfischer-no

Reply:
I think k_work_submit should actually be `k_work_submit_to_queue(&dwt_work_queue, &ctx->irq_cb_work);` I will look through my WIP branches.